specs/ExtendedAcceleratorKeyPressed.md

```diff
@@ -0,0 +1,187 @@
+# Background
+WebView2 has `ICoreWebView2AcceleratorKeyPressedEventArgs` for the 
+`CoreWebView2Controller.AcceleratorKeyPressed` event. We have been asked to extend this API so
+that developers can indicate that theywould like an existing browser accelerator key to be 

```

Comment:
There's a main difference between this new property and Handled property that is explained in the API's comment section below. I will update this description to include that information to avoid confusion.

Reply:
updated!